### PR TITLE
change readme, repo structure and add kubernetes example

### PR DESCRIPTION
**Pull request by AuPath**
- Changed repo structure
- Added kubernetes example, removed the 'metrics' component as it is not relevant
- Slimmed down the main README file

### CODE REVIEW

File: Kubernetes/camel-context.xml
Comment (marioscrock) [inline Kubernetes/camel-context.xml:1]: Remove this file

File: README.md
Comment (marioscrock) [inline README.md:1]: Specify that the Dockerfiles provided should be modified to be executed considering a Chimera project

File: Kubernetes/Dockerfile
Comment (marioscrock) [inline Kubernetes/Dockerfile:1]: Change also this one to be generic and not related to "chimera-tutorial.jar"

File: Kubernetes/README.md
Comment (marioscrock) [inline Kubernetes/README.md:1]: Change to be generic (e.g., remove mention to cefriel/chimera:tutorial-advanced) and perform a check with a different project to see if it works.

File: Kubernetes/chimera-converter.yml
Comment (marioscrock) [inline Kubernetes/chimera-converter.yml:1]: Change it to be generic not bound to chimera-tutorial-advanced

**Review by marioscrock — CHANGES_REQUESTED** [review]

File: README.md
Comment (marioscrock) [reply README.md:1]: Maybe we can add something from the README prepared by Gianluca. At least briefly explain the artifacts in each folder. Alternatively we can split it in 3 and put three README in the 3 folders.

Comment (AuPath) [reply README.md:1]: I added a readme to each deployment option

File: Kubernetes/chimera-converter.yml
Comment (marioscrock) [inline Kubernetes/chimera-converter.yml:1]: Add also example file initialised to execute chimera-minimal? Can be a good idea also to make "runnable" the other deployment templates.

**Review by marioscrock — COMMENTED** [review]
Add also example file initialised to execute chimera-minimal? Can be a good idea also to make "runnable" the other deployment templates.

**Pull request merged**